console-src/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "backstage",
```

Comment:
nit: let's use sth. like `console` here

Reply:
good idea 🙂 I have used `api-console` as `console` being a core node module, it was raising a warning: 0688738bfb6fa7e6db3a8c1f58c00f710cf9eeb1